dockers/broad/samtools/docker_build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+set -e
+
+# Update version when changes to Dockerfile are made
+DOCKER_IMAGE_VERSION=1.0.0
+TIMESTAMP=$(date +"%s")
+DIR=$(cd $(dirname $0) && pwd)
+
+# Registries and tags
+GCR_URL="us.gcr.io/broad-gotc-prod/samtools"
+IMAGE_TAG="$DOCKER_IMAGE_VERSION-$TIMESTAMP"
```

Comment:
I think this should be the samtools version primarily.  Probably important to have the docker image version here too i guess.

Reply:
yeah i think thats totally reasonable, very easy fix!